api.yaml

```diff
@@ -779,6 +809,9 @@ components:
         is_refundable:
           type: boolean
           description: Indicates whether the product supports cancellations or not.
+        provides_pricing:
+          type: boolean
+          description: Indicates whether the product supports pricing or not.
```

Comment:
**suggestion:** IMHO It would be more clear if instead of `supports` says `contains`